packages/@okta/vuepress-theme-prose/components/Hero.vue

```diff
@@ -1,12 +1,29 @@
 <template>
-  <section class="hero">
+  <!-- Remove style attribute from .hero section when removing promo banner -->
+  <section
+    class="hero"
+    style="padding-top: 28px; padding-bottom: 28px;"
+  >
     <div class="hero__wrapper wrapper">
       <h1 class="hero__title dont-break-out">
         Okta developer
       </h1>
       <p class="hero__paragraph dont-break-out">
         Our developer portal enables you to deploy auth that protects your users, apps, APIs, and infrastructure.
       </p>
+      <!-- Start workshops & Oktane promo banner -->
+      <p style="color: white; font-size: 18px; line-height: 1.4; max-width: 500px;">
+        Get your app <a
+          style="color: white; text-decoration: underline;"
+          href="https://regionalevents.okta.com/enterprisereadyworkshops"
+          target="_blank"
+        >enterprise-ready with free virtual workshops</a>, then <a
```

Comment:
Would it be better to just add the link on `free virtual workshops` instead of `enterprise-ready with free virtual workshops`?

Reply:
SEO stuff makes sense. LGTM.